src/Exceptionless.Web/Controllers/AuthController.cs

```diff
@@ -247,7 +247,12 @@ public async Task<ActionResult<TokenResult>> SignupAsync(Signup model)
             EmailAddress = email,
             IsEmailAddressVerified = _authOptions.EnableActiveDirectoryAuth
         };
-        user.CreateVerifyEmailAddressToken();
+
+        if (user.IsEmailAddressVerified)
```

Comment:
We are creating the user right above. How is it possible for them to be verified already?

Reply:
It's based on the ad configuration. Safer to do this to ensure correct state